train.py

```diff
@@ -170,6 +170,18 @@ def validate_directories(args):
     }
 
 
+def check_sample_size(dilations, sample_size):
+    '''Check that the sample size is greater than the receptive field size and
+       raise an exception if not.'''
+
+    receptive_field_size = compute_receptive_field_size(dilations)
+    if sample_size is not None:
+        if sample_size <= receptive_field_size:
+            raise ValueError('Sample size {} must be greater than the '
+                             'receptive field size {}.'. \
```

Comment:
Why single quotes? Is it to be consistent with the rest of the file? If that's the case, let me know and I'll fix it in the PR as well.